packages/grafana-ui/src/components/DataLinks/DataLinksContextMenu.tsx

```diff
@@ -18,8 +20,25 @@ export interface DataLinksContextMenuApi {
 
 export const DataLinksContextMenu: React.FC<DataLinksContextMenuProps> = ({ children, links, config }) => {
   const linksCounter = config.links!.length;
-  const getDataLinksContextMenuItems = () => {
-    return [{ items: linkModelToContextMenuItems(links), label: 'Data links' }];
+  const itemsGroup: MenuItemsGroup[] = [];
+  itemsGroup.push({ items: linkModelToContextMenuItems(links), label: 'Data links' });
```

Comment:
```suggestion
  const itemsGroup: MenuItemsGroup[] = [{ items: linkModelToContextMenuItems(links), label: 'Data links' }];
```
You can initialise `itemsGroup` with `items`, no need to push them separately.

Reply:
I just learned something new today 😄
Thanks! @Clarity-89  👍